lib/ansible/plugins/inventory/gcp_compute.py

```diff
@@ -50,30 +50,38 @@
             choices: ['application', 'serviceaccount', 'machineaccount']
             env:
                 - name: GCP_AUTH_KIND
-                  version_added: "2.8"
+                  version_added: "2.8.2"
         scopes:
             description: list of authentication scopes
             type: list
             default: ['https://www.googleapis.com/auth/compute']
             env:
                 - name: GCP_SCOPES
-                  version_added: "2.8"
+                  version_added: "2.8.2"
         service_account_file:
             description:
                 - The path of a Service Account JSON file if serviceaccount is selected as type.
             type: path
             env:
                 - name: GCP_SERVICE_ACCOUNT_FILE
-                  version_added: "2.8"
+                  version_added: "2.8.2"
                 - name: GCE_CREDENTIALS_FILE_PATH
                   version_added: "2.8"
+        service_account_contents:
```

Comment:
Shouldn't the whole new option be marked as `version_added: "2.8.2"`? (Sorry I missed that earlier...)

Reply:
`GCE_CREDENTIALS_FILE_PATH` was actually added in 2.8.0, so the current docs are correct :)

Reply:
But the option `service_account_contents` itself was not there before. And it does not say that it was added in 2.8.2. Or is there no `version_added` for options?

Reply:
Ah, now I see. What's the best way to fix it now?

Reply:
I guess creating another PR (for `devel`, including the version numbers added here), and backport it. @abadger wanted to do another round of backport merging today, so if you do it now, it should still work out.